internal/blocksync/reactor.go

```diff
@@ -503,8 +506,17 @@ func (bcR *Reactor) isMissingExtension(state sm.State, blocksSynced uint64) bool
 	return missingExtension
 }
 
+func (bcR *Reactor) localNodeBlocksTheChain(state sm.State) bool {
+	_, val := state.Validators.GetByAddress(bcR.myAddr)
+	if val == nil {
+		return false
+	}
+	total := state.Validators.TotalVotingPower()
+	return val.VotingPower > total*2/3
```

Comment:
Wait a minute... I think this should be 1/3 (!)
@cason  ?

Reply:
This is a good questions, and depends on what behavior we want here:

1. The validator runs network by itself, in other words, no other validator is needed for progress: `val.VotingPower > total*2/3`
2. The network cannot run without this validator, in other words, this validator is required for progress: `val.VotingPower >= total/3`

Reply:
From the name of the method, I would say 2.

Because if the validator is needed for progress and it crashes, no new block was produced since its crash, therefore block sync is useless. Do you agree?

Reply:
After some internal discussion @cason and I have settled on option 2. Commit coming up